packages/wallets/src/components/WalletsErrorScreen/WalletsErrorScreen.tsx

```diff
@@ -1,27 +1,43 @@
-import React from 'react';
-import { LegacyWarningIcon } from '@deriv/quill-icons';
+import React, { ComponentProps } from 'react';
+import useDevice from '../../hooks/useDevice';
+import ErrorIcon from '../../public/images/error-icon.svg';
 import { WalletButton } from '../Base';
 import { WalletsActionScreen } from '../WalletsActionScreen';
 
 type TProps = {
+    buttonText?: string;
+    buttonVariant?: ComponentProps<typeof WalletButton>['variant'];
     message?: string;
+    onClick?: () => void;
+    title?: string;
 };
 
-const ErrorState: React.FC<TProps> = ({
+const WalletsErrorScreen: React.FC<TProps> = ({
+    buttonText = 'Try again',
+    buttonVariant = 'ghost',
     message = 'Sorry an error occurred. Please try accessing our cashier page again.',
+    onClick,
+    title = 'Oops, something went wrong!',
 }) => {
+    const { isMobile } = useDevice();
+    const imageSize = {
+        height: isMobile ? 80 : 128,
+        width: isMobile ? 80 : 128,
+    };
+
     return (
         <WalletsActionScreen
             description={message}
-            icon={<LegacyWarningIcon data-testid='dt_error_icon' fill='#FF444F' iconSize='2xl' />}
+            //TODO: replace with quill-icons
+            icon={<ErrorIcon {...imageSize} data-testid='dt_error_icon' />}
```

Comment:
Can't we use `DerivLightErrorIconIcon` from quill-icons for this?

Reply:
oh, yes sure, i missed this icon

Reply:
sorted